javascript/javascript3/week2/movies.js

```diff
@@ -0,0 +1,10 @@
+// Movies exercise  
+
+fetch('https://gist.githubusercontent.com/pankaj28843/08f397fcea7c760a99206bcb0ae8d0a4/raw/02d8bc9ec9a73e463b13c44df77a87255def5ab9/movies.json')
+.then(response=>response.json())
+.then(movies=>{    
+const badMovies = movies.filter(movie=>movie.rating<=3);
```

Comment:
The code is correct but the formatting is not clean, makes it hard to read. Consider installing the "prettier" extension for VSCode